src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -961,19 +1019,59 @@ private void Submit()
 
                 Package.AddAssemblies(Assemblies);
 
-                // begin submission
-                var handle = pmExtension.PackageManagerClient.PublishAsync(Package, files, IsNewVersion);
-
-                // start upload
-                Uploading = true;
-                UploadHandle = handle;
-
+                return files;
             }
             catch (Exception e)
             {
                 ErrorString = e.Message;
                 dynamoViewModel.Model.Logger.Log(e);
             }
+
+            return null;
```

Comment:
API design guidelines suggest that an empty list should be return from an API that returns a list, rather than returning a `null`. Maybe `return new string[] { };` or something like that.